### DISCUSSION TITLE
Window moves to background when running a query.

### DISCUSSION THREAD

**Discussion by rob-comparitec** [Sequel-Ace/Sequel-Ace#1694] [General]
It seems after a recent update, every time a query is executed (click or keyboard shortcut), the sequel ace window moves to the background and I have to toggle it (alt + tab it) back to the front. This is not a machine specific issue as everyone else in our team who has upgraded to this version also has this issue. We are all using M1 or M2 silicon chip Macs. 

Our version is 4.0.3 > Build 20042.

**Comment by Jason-Morcos** [comment]
https://github.com/Sequel-Ace/Sequel-Ace/issues/1618
